dolphinscheduler-e2e/dolphinscheduler-e2e-case/src/test/java/org/apache/dolphinscheduler/e2e/pages/project/workflow/TaskInstanceTab.java

```diff
@@ -35,7 +35,8 @@
 
 @Getter
 public final class TaskInstanceTab extends NavBarPage implements ProjectDetailPage.Tab {
-    @FindBy(className = "items-task-instances")
+
+    @FindBy(css = "tbody > .n-data-table-tr")
```

Comment:
Please avoid too wide css selector. Id or classname is preferred. If not, it is recommended to use the xpath selector to specify the scope selection, otherwise we may encounter `stale element` exception during the page jumping.

Reply:
The form doesn't have id, and it's hard to use xpath since all class is not unique
![Uploading image.png…]()

Reply:
I add class in this form